977.squares-of-a-sorted-array.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * URL of this problem
+ * https://leetcode.com/problems/squares-of-a-sorted-array/
+ */
+
+/**
+ * @param {number[]} nums
+ * @return {number[]}
+ */
+var sortedSquares = function(nums) {
+    
+    let result = [];
+    
+    //Calculate area
+    for (let i = 0; i < nums.length; i++) {
+        const area = Math.abs(nums[i] * nums[i]);
```

Comment:
```suggestion
        const square = nums[i] * nums[i];
```
I think it is better to use "square" instead of "area"
You can calculate the squares without Math.abs()